### DISCUSSION TITLE
Не работают сервисы Microsoft

### DISCUSSION THREAD

**Discussion by acidlame** [bol-van/zapret#806] [Q&A]
Когда включен запрет все работает отлично, кроме ЛЮБЫХ сервисов от Microsoft. Что Microsoft store, что приложение xbox. Самое странное что дыже перестают работать скины в майнкрафт. Причем как лицензия так и даже пиратские сервисы по типу ely.by . НА серверах вообще почти все звуки пропадают. Как это все можно пофиксить, вдруг кто сталкивался?

**Comment by toonbr1me** [comment]
случайно у тебя не кастомный dns? comss, например

**Reply by brnsky** [reply]
А у тебя с comss дохнет что-то еще? У меня, например, с комсс днс отваливаться ютуб начал с включенным zapret, хотя до этого проблем не было

**Reply by toonbr1me** [reply]
> А у тебя с comss дохнет что-то еще? У меня, например, с комсс днс отваливаться ютуб начал с включенным zapret, хотя до этого проблем не было

бывает время от времени что-то да упадёт, не особо контролируемый процесс
